java/code/src/com/redhat/rhn/domain/server/NetworkInterface.java

```diff
@@ -313,6 +313,14 @@ public boolean isBond() {
         return "bonding".equals(module);
     }
 
+    /**
+     * true if the nic is a container network
+     * @return true if the nic is a container network
+     */
+    public boolean isContainerNetwork() {
+        return ("bridge".equals(module) && (getName().startsWith("docker") || getName().startsWith("cni-podman")));
```

Comment:
Is there any other way to do it?  I see 2 problems:
- `netavark` is missing (but probably you just need to add `getName().startsWith("cni-netavark")`
-  I don't think this would work for K3s (I know, it's not released yet, but we're trying to do our best to not introduce something that might work for podman and not for kubernetes)

Reply:
Thanks for the review @mbussolotto !

Ok, good point about `netavark` and Kubernetes. I'll try to enhance the mechanism.

In the end, I just mimic here what we already do for the `virbr*` devices: https://github.com/uyuni-project/uyuni/pull/9057/files#diff-dd63c17a089292848e40dfb8c02de48edb6b619425601d9fe81d02aa2cf3ba5eR329